File: 백준/Silver/13909. 창문 닫기/창문 닫기.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;

public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int n = Integer.parseInt(br.readLine());

        int count = 1;
        for(int i = 2; i * i <= n; i++){
            count ++;
        }
        StringBuilder sb = new StringBuilder();
        sb.append(count).append("\n");

        bw.write(sb.toString());
        bw.flush();
        bw.close();
    }
}

File: 백준/Silver/13909. 창문 닫기/창문 닫기.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import kotlin.math.sqrt

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val bw = BufferedWriter(OutputStreamWriter(System.out))

    val n = br.readLine().toInt()
    var count = 1
    for(i in 2..sqrt(n.toDouble()).toInt()) {
        count++
    }

    val sb = StringBuilder()
    sb.append(count)

    bw.write(sb.toString())
    bw.flush()
    bw.close()
}